app/src/main/java/fr/free/nrw/commons/media/Caption.java

```diff
@@ -0,0 +1,40 @@
+package fr.free.nrw.commons.media;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Model class for parsing Captions when fetching captions using filename in MediaClient
+ */
+public class Caption {
+
+    @SerializedName("language")
+    private String language;
+    @SerializedName("value")
+    private String value;
+
+    /**
+     * No args constructor for use in serialization
+     */
+    public Caption() {
+    }
+
+    /**
+     * @param value
+     * @param language
+     */
+    public Caption(String language, String value) {
+        super();
+        this.language = language;
+        this.value = value;
+    }
+
+    @SerializedName("language")
```

Comment:
simple javadoc such as "Human language in which the caption is written" for all methods

Reply:
done